peerscout_api/recomend_editor.py

```diff
@@ -1,73 +1,66 @@
 import os
 import logging
-from typing import List, Tuple, T
+from typing import List, NamedTuple, Tuple, T
+from numpy import array
 import pandas as pd
 
 from sklearn.metrics.pairwise import cosine_similarity
 
 from data_science_pipeline.utils.io import load_object_from
 from data_science_pipeline.peerscout.models import (
+    Vectorizer,
     WeightedKeywordModel
 )
 
 LOGGER = logging.getLogger(__name__)
 
 
-def load_model(model_path: str, model_name: str) -> dict:
-    model_full_path = os.path.join(model_path, model_name)
-    LOGGER.info('Loading model : %s', model_full_path)
-    model_dict = load_object_from(model_full_path)
-
-    return model_dict
-
-
-def get_editor_tf_idf_vectorizer(model_dict: dict):
-    return model_dict['editor_tf_idf_vectorizer']
-
-
-def get_editor_tf_idf(model_dict: dict):
-    return model_dict['editor_tf_idf']
+class PeerScoutModelProps(NamedTuple):
+    editor_names: List[str]
+    editor_person_ids: List[str]
+    editor_tf_idf_vectorizer: Vectorizer
+    editor_tf_idf: array
```

Comment:
I actually made the mistake before. The array type in numpy seems to be `ndarray`, whereas array is a function.
(I would actually use `np.ndarray` to make it more visible where it is coming from)